Bottom: dc25ff41bac64690c5b1903317bb391408a28cbe
Top:    1dd51c5d0753d8cdb0aff2c65bf5c72b83607322
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 19:17:12 +0100

Refresh of fix-fullpoly-flag-test

---

diff --git a/src/action.c b/src/action.c
index 92fdbae..6b979d3 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1342,7 +1342,7 @@ NotifyMode (void)
 	  PolygonTypePtr polygon;
 
 	  int flags = CLEARPOLYFLAG;
-	  if (Settings.FullPoly)
+	  if (TEST_FLAG (NEWFULLPOLYFLAG, PCB))
 	    flags |= FULLPOLYFLAG;
 	  if ((polygon = CreateNewPolygonFromRectangle (CURRENT,
 							Crosshair.
